_about_livecoms/04_donations.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: single
+sidebar:
+  nav: donations.md
+title: Donating to LiveCoMS 
+excerpt: How to donate to LiveCoMS
+permalink: /about/donations
+---
+
+### Donations to LiveCoMS
+
+  We very much appreciate financial donations to LiveCoMS! Donations
+  are made to the CU Foundation of University of Colorado Boulder, the
+  official publisher of the journal, to a fund exclusively for use of
+  LiveCoMS and controlled by the managing editors. The CU Foundation
+  is a 501(c)(3) charitable organization, and donations are thus
+  usually tax-deductible.
+ 
+Follow the link felow to donate:
+
+  <p><a href="https://giving.cu.edu/fund/living-journal-computational-molecular-science-support-fund" class="btn ">Donate Now to LiveCoMS!</a></p>
+
+You can also support LiveCoMS by:
+  * Contributing to LiveCoMS articles.
+  * Expressing your interest in becoming a reviewer ([early career](/about/earlycareer) or otherwise).
```

Comment:
Maybe note that currently we have a large backlog of such reviewers? I don't think we want to encourage a deluge of them at present, becuase someone has to track and already we have a lot more than we're using.

Reply:
I would hesitate to discourage potential reviwers.  There's not much harm in having a bigger list - to us or the reviewers ... we don't promise anything.

Reply:
agreed with dan.